### PR TITLE
Bpk 4276 restructure remaining f pages

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: docs/src/layouts/links.js [hunk 1]
```diff
@@ -148,7 +148,7 @@ const ComponentsLinks = [
       {
         id: 'FORM_VALIDATION',
         route: routes.FORM_VALIDATION,
-        children: 'Validation',
+        children: 'Form validation',
```
Comment (georgegillams) [inline docs/src/layouts/links.js:151]: This just renames the page in the side-bar. I thought it makes more sense for it to live with Form elements instead of being right at the bottom of the list

File: docs/src/pages/IconPage/IconPage.js [hunk 2]
```diff
@@ -36,7 +36,9 @@ const Page = () => (
       <IntroBlurb>
         A suite of icons for representing concepts, features and actions.
         Looking for the icon set? Check out the{' '}
-        <BpkLink href="/components/icon?platform=design">design tab</BpkLink>{' '}
+        <BpkRouterLink to="/components/icon?platform=design">
+          design tab
+        </BpkRouterLink>{' '}
```
Comment (georgegillams) [inline docs/src/pages/IconPage/IconPage.js:41]: Unrelated change - this just saves the entire page reloading when the link it used

File: docs/src/pages/FlatListPage/FlatListPage.js [hunk 3]
```diff
@@ -16,6 +16,26 @@
  * limitations under the License.
  */
 
-import page from './AndroidFlarePage';
```
Comment (olliecurtis) [inline docs/src/pages/FlatListPage/FlatListPage.js:19]: Is this page renaming and change expected seems a bit strange going from a Flare page to FlatListPage?

File: docs/src/pages/FlatListPage/index.js [hunk 4]
```diff
@@ -16,6 +16,6 @@
  * limitations under the License.
  */
 
-import page from './IOSFlarePage';
```
Comment (olliecurtis) [inline docs/src/pages/FlatListPage/index.js:19]: Same again here

File: docs/src/routes/Routes.js [hunk 5]
```diff
@@ -87,7 +87,7 @@ import MapPage from '../pages/MapPage';
 import InfiniteScrollPage from '../pages/InfiniteScrollPage';
 import MobileScrollContainerPage from '../pages/MobileScrollContainerPage';
 import ModalsPage from '../pages/ModalsPage';
-import NativeFlatListPage from '../pages/NativeFlatListPage';
+import NativeFlatListPage from '../pages/FlatListPage';
```
Comment (olliecurtis) [inline docs/src/routes/Routes.js:90]: If we are renaming this should the import also be FlatListPage to be consistent?

**Review by olliecurtis — CHANGES_REQUESTED** [review]
Could you just double check some weird behaviour with pages changing to make sure its ok? Will run locally also to check

File: docs/src/pages/FlatListPage/FlatListPage.js (hunk 3 above)
Comment (georgegillams) [reply docs/src/pages/FlatListPage/FlatListPage.js:19]: I think this is just git being dumb and thinking that I renamed this file when actually I deleted it and added another

Comment (olliecurtis) [reply docs/src/pages/FlatListPage/FlatListPage.js:19]: 👍

File: docs/src/routes/Routes.js (hunk 5 above)
Comment (georgegillams) [reply docs/src/routes/Routes.js:90]: Yes

**Review by olliecurtis — APPROVED** [review]
